CMSgov/mint-app
# [NOREF] HelpScout POC

# NOREF

## Changes and Description

- Enables HelpScout beacon in MINT

There are currently no articles or content written for the beacon. The main intent of this PR is to introduce the code to neatly initialize and destroy the beacon when users are either not authenticated OR have the feature flag turned off.

## How to test this change

- `scripts/dev up`
- Sign in with Okta
- Enable the `helpScoutEnabled` feature flag for your user in LaunchDarkly's "local" env
- Ensure you see the beacon only when signed in

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
